client/src/components/mapping/InputGroup/index.tsx

```diff
@@ -53,35 +79,87 @@ const InputGroup = ({ inputGroup }: Props) => {
   };
 
   return (
-    <React.Fragment>
-      {inputGroup.conditions.map(
-        (condition: any, index: number) =>
-          condition && <InputCondition key={index} condition={condition} />
-      )}
-      <Button
-        icon={'add'}
-        onClick={(e: React.MouseEvent) => {
-          e.stopPropagation();
-          createCondition({
-            variables: {
-              inputGroupId: inputGroup.id
-            }
-          });
-        }}
-      >
-        Add condition
-      </Button>
-      <div id="input-columns">
-        <div id="input-column-rows">
-          {inputGroup.inputs.map(
-            (input: any, index: number) =>
-              input && <InputColumn key={index} input={input} />
-          )}
+    <Card key={key} elevation={Elevation.ONE}>
+      <div className="delete-input-group">
+        <Button
+          icon="trash"
+          intent="danger"
+          minimal={true}
+          onClick={() => {
```

Comment:
inline function

Reply:
What do you mean?

Reply:
remove bracket

Reply:
Doesn't this mean that the result of `deleteInputGroup` will be returned? We don't really care but still 🤷‍♂️

Reply:
yes it would return the result but as you said we don't care
Although if you find it confusing it's probably not the best idea so leave it as it is :+1: